### ISSUE TITLE
Support Subresource Integrity

### ISSUE THREAD

**Issue by guybedford** [systemjs/systemjs#639]
Something like the following:

```javascript
System.config({
  meta: {
    'some/resource.js': {
      integrity: 'sha256-...',
    }
  }
});
```

It would only work for script loads initially of course so we should throw an error when a resource has an `integrity` property but is not script-loaded. Plugins could also do what they want with it to make it work for other tags. The fetch specification would support this for fetching in future so eventually it would be everywhere.

**Comment by matthewbauer** [comment]
While you're at it, might make sense to add "nonce" support as well.

**Comment by guybedford** [comment]
Released in 0.18.11.
